### PR TITLE
Workshop #7401 - Updated the manfest.json files.

### PR DESCRIPTION
**Pull request by SuRajan**
updated the manifest.json files in freetier and livelabs folders.
Workshop #7401

### CODE REVIEW

File: data-management-library/oracle-machine-learning/oml-fun-on-ADB/omlserv/omlserv.md [hunk 1]
```diff
@@ -1,22 +1,39 @@
-# Title of the Lab
```
Comment (rfontcha) [inline data-management-library/oracle-machine-learning/oml-fun-on-ADB/omlserv/omlserv.md:1]: @SuRajan - Please remove changes to this file from your request. It's not part of WMS ID 7041

**Review by rfontcha — CHANGES_REQUESTED** [review]
@SuRajan - View comment about unrelated file to be removed from your PR

**Review by rfontcha — CHANGES_REQUESTED** [review]
@SuRajan - Actually this entire PR has nothing to do with 7041. Please rename the title accordingly with the correct WMS ID. If the workshop is not yet in WMS, then withdraw your pull request and first submit the workshop entry to WMS

**Comment by SuRajan** [comment]
@rfontcha This PR is for workshop #7401(OML Fundamentals on ADB). Are you perhaps confusing this with #7041?
Please let me know.

**Review by rfontcha — APPROVED** [review]
@SuRajan Thanks for pointing that out. My mistake, very easy to confuse as they look alike 7401 vs 7041!

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**